lms/djangoapps/grades/new/course_grade.py

```diff
@@ -31,11 +31,15 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
-        self.course_version = getattr(course, 'course_version', None)
-        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
-        self.course_structure = course_structure
         self._percent = None
         self._letter_grade = None
+
+        self.course_structure = course_structure
+        if self.course_structure:
```

Comment:
Without this change, the following code path was breaking: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/new/course_grade.py#L258

Essentially, we have code that doesn't require data from the course_structure as it is read from SQL tables instead.  I have refactored the code (to be merged in a separate PR) so this is much cleaner.